src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder()
```

Comment:
let's create a variable of type `Plane` and call `build()` here as well in the end